project/frontend/src/components/ButtonGroup.js

```diff
@@ -9,11 +9,38 @@ import React from 'react';
  */
 function ButtonGroup(props) {
   const { labelList } = props;
-  const buttonList = [];
-  for (let delta = 0; delta < labelList.length; delta++) {
-    buttonList.push(<button>{labelList[delta]}</button>);
-  }
-  return <div className='btn-group'>{buttonList}</div>;
+  const [backgroundIsGray, setBackgroundIsGray] = useState({
```

Comment:
Maybe we could rename this to `isSelected` or something (it looks to me like that is what backgroundIsGray indicates but correct me if I'm wrong!) just so it is clearer what is happening